docs/architecture.md

```diff
@@ -15,9 +15,9 @@ _Some incomplete notes_
 
 ### Audit/Report terminology
 * **Category** - Roll-up collection of audits and audit groups into a user-facing section of the report (eg. `Best Practices`). Applies weighting and overall scoring to the section. Examples: PWA, Accessibility, Best Practices.
-* **Audit description** - Short user-visible title for the successful audit. eg. “All image elements have `[alt]` attributes.”
-* **Audit failure description** - Short user-visible title for a failing  audit. eg. “Some image elements do not have `[alt]` attributes.”
-* **Audit help text** - Explanation of why the user should care about the audit. Not necessarily how to fix it, unless there is no external link that explains it. ([See helpText guidelines](CONTRIBUTING.md#helptext-guidelines)). eg. “Informative elements should aim for short, descriptive alternate text. Decorative elements can be ignored with an empty alt attribute. [Learn more].”
+* **Audit title** - Short user-visible title for the successful audit. eg. “All image elements have `[alt]` attributes.”
+* **Audit failure title** - Short user-visible title for a failing  audit. eg. “Some image elements do not have `[alt]` attributes.”
```

Comment:
should we change it to `**Audit failureTitle**` just to be crystal clear it's that property name?

Reply:
done